modules/apps/frontend-data-set/frontend-data-set-impl/src/main/java/com/liferay/frontend/data/set/internal/resolver/FDSRestApplicationURLParameterResolverRegistryImpl.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.frontend.data.set.internal.resolver;
+
+import com.liferay.frontend.data.set.resolver.FDSRestApplicationURLParameterResolver;
+import com.liferay.frontend.data.set.resolver.FDSRestApplicationURLParameterResolverRegistry;
+import com.liferay.osgi.service.tracker.collections.map.ServiceTrackerCustomizerFactory;
+import com.liferay.osgi.service.tracker.collections.map.ServiceTrackerCustomizerFactory.ServiceWrapper;
+import com.liferay.osgi.service.tracker.collections.map.ServiceTrackerMap;
+import com.liferay.osgi.service.tracker.collections.map.ServiceTrackerMapFactory;
+import com.liferay.petra.string.StringBundler;
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.kernel.util.ListUtil;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.osgi.framework.BundleContext;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Deactivate;
+
+/**
+ * @author Gianmarco Brunialti Masera
+ */
+@Component(service = FDSRestApplicationURLParameterResolverRegistry.class)
+public class FDSRestApplicationURLParameterResolverRegistryImpl
+	implements FDSRestApplicationURLParameterResolverRegistry {
+
+	@Override
+	public FDSRestApplicationURLParameterResolver getResolver(
+		String restApplication, String restSchema) {
+
+		String resolverKey = StringBundler.concat(
+			restApplication, "/", restSchema);
```

Comment:
This is good enough for now, but later on we probably want to extend this to include `fdsName` argument. The current solution implies that endpoints of an application schema are resolved in the same way. While they are used in only one context, like commerce context in this PR, this is fine. It is for sure the most common way to use endpoints. But if we or a customer wants to use the same application / schema in multiple contexts, one resolver will override the other. cc @dsanz @herrardo